system/sched_note/note_main.c

```diff
@@ -457,6 +457,7 @@ static void dump_notes(size_t nread)
             {
               FAR struct note_spinlock_s *note_spinlock =
                 (FAR struct note_spinlock_s *)note;
+              FAR void *spinlock;
```

Comment:
BTW, @YuuichiNakamura could you update note_main.c to support syscall/irq you add recently?

Reply:
@xiaoxiang781216 These are already supported by note_main.c line 592-671.